src/components/SignInSignUpModal/RegisterForm/RegisterForm.tsx

```diff
@@ -35,8 +34,10 @@ type LoginProps = {
 function RegisterForm({ manageRequestMessage }: RegisterFormProps) {
   const login: LoginProps = useContext(LoginContext);
   const [values, setValues] = useState({
+    name: '',
     password: '',
     email: '',
+    phone: '',
     showPassword: false,
   });
```

Comment:
Since you are using react-hook-form I would recommend using useForm hook with defaultValues object, much easier to control the output and perform further steps. They have pretty neat docs with nice explanatory video here: https://react-hook-form.com/get-started and also on other pages You can find examples of using it with TS.